docs/guides/tanzu-mission-control_policy.md

```diff
@@ -278,5 +278,82 @@ resource "tanzu-mission-control_custom_policy" "cluster_group_scoped_tmc-block-r
 ## Custom Template and Custom Policy
```

Comment:
Section  *Custom Template and Custom Policy* seems to only have example for creating custom policy template. But not usage of said custom template in a custom policy.

And perhaps, section *Custom Policy on a Cluster Group* should be under this section? As an example of applying custom policy via Terraform.